Scripts/Runtime/Entities/TaskSystem/AbstractTaskDriver.cs

```diff
@@ -78,6 +78,8 @@ protected AbstractTaskDriver(World world, AbstractTaskSystem abstractTaskSystem)
             
             TaskStreamFactory.CreateTaskStreams(this, TaskStreams);
             CancelRequestsDataStream = new CancelRequestsDataStream();
+
+            TaskDriverFactory.CreateSubTaskDrivers(this, m_SubTaskDrivers, world);
```

Comment:
In a previous PR I talked about potentially just grabbing world from the system passed in.
If that was a good suggestion the same goes here. The World may be read off of the `AbstractTaskDriver`

Reply:
It would be a good suggestion if `World.GetOrCreateSystem` actually gave the System it's World reference. But unfortunately it only constructs the instance and we still have to wait till later for the `OnCreate` to be called and the system to get a World reference.

Reply:
Changed the API here since you are right, we can get the World off the passed in TaskDriver